src/sidebar/components/test/tag-editor-test.js

```diff
@@ -81,6 +82,10 @@ describe('TagEditor', function() {
       store: sinon.stub(),
     };
     $imports.$mock(mockImportedComponents());
+    // create a full dom tree for a11y testing
+    $imports.$mock({
+      './autocomplete-list': AutocompleteList,
```

Comment:
This works, although it also affects all tests, whereas we really only need to make this change for the accessibility tests. To do that, you can put this logic in the `content` function for a11y tests, or the `beforeEach` function if the a11y tests are grouped in a `describe` block.

Using `$imports.$mock` to undo a mock like this works, although if you update the `babel-plugin-mockable-imports` package to the latest version there is [an easier/more general method](https://github.com/robertknight/babel-plugin-mockable-imports#restoring-specific-mocks):

```js
$imports.$restore({
 './autocomplete-list': true,
});
```

Reply:
awe, good call.